src/main/webapp/index.html

```diff
@@ -7,7 +7,7 @@
     <title>smallcity</title>
   </head>
   <!-- TODO: Move getGeolocation() function call to sensical location -->
-  <body onload="getGeolocation()">
+  <body onload="getGeolocation(); displayList()">
```

Comment:
The Location of the display function (and getGeolocation function) will likely have to move to make sure that we do not try to display any listings before we have the user's location. I'll create an issue to track this

Reply:
Gotcha